packages/react-components/src/components/SendBox.tsx

```diff
@@ -199,40 +220,48 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   );
 
   return (
-    <InputBoxComponent
-      autoFocus={autoFocus}
-      data-ui-id={ids.sendboxTextField}
-      inlineChildren={true}
-      disabled={disabled}
-      errorMessage={onRenderSystemMessage ? onRenderSystemMessage(errorMessage) : errorMessage}
-      textFieldRef={sendTextFieldRef}
-      id="sendbox"
-      inputClassName={sendBoxStyle}
-      placeholderText={strings.placeholderText}
-      textValue={textValue}
-      onChange={setText}
-      onKeyDown={() => {
-        onTyping && onTyping();
-      }}
-      onEnterKeyDown={() => {
-        sendMessageOnClick();
-      }}
-      styles={mergedStyles}
-      supportNewline={supportNewline}
-      maxLength={MAXIMUM_LENGTH_OF_MESSAGE}
-    >
-      <InputBoxButton
-        onRenderIcon={onRenderSendIcon}
-        onClick={(e) => {
-          if (!textValueOverflow) {
-            sendMessageOnClick();
-          }
-          e.stopPropagation();
+    <Stack className={mergeStyles(borderAndBoxShadowStyle, sendBoxStyleMain)}>
+      <InputBoxComponent
+        autoFocus={autoFocus}
+        data-ui-id={ids.sendboxTextField}
+        inlineChildren={true}
+        disabled={disabled}
+        errorMessage={onRenderSystemMessage ? onRenderSystemMessage(errorMessage) : errorMessage}
+        textFieldRef={sendTextFieldRef}
+        id="sendbox"
+        inputClassName={sendBoxStyle}
+        placeholderText={strings.placeholderText}
+        textValue={textValue}
+        onChange={setText}
+        onKeyDown={() => {
+          onTyping && onTyping();
+        }}
+        onEnterKeyDown={() => {
+          sendMessageOnClick();
         }}
-        id={'sendIconWrapper'}
-        className={mergedSendButtonStyle}
-        ariaLabel={localeStrings.sendButtonAriaLabel}
-      />
-    </InputBoxComponent>
+        styles={mergedStyles}
+        supportNewline={supportNewline}
+        maxLength={MAXIMUM_LENGTH_OF_MESSAGE}
+      >
+        <InputBoxButton
+          onRenderIcon={onRenderSendIcon}
+          onClick={(e) => {
+            if (!textValueOverflow) {
+              sendMessageOnClick();
+            }
+            e.stopPropagation();
+          }}
+          id={'sendIconWrapper'}
+          className={mergedSendButtonStyle}
+          ariaLabel={localeStrings.sendButtonAriaLabel}
+        />
+      </InputBoxComponent>
+      <Stack className={fileCardBoxStyle}>
```

Comment:
This stack should also be rendered conditionally as it is adding unnecessary padding to the sendbox even in the stable builds.

Reply:
Done